### ISSUE TITLE
Fountains are using 13% CPU Usage (just 1 fountain already)

### ISSUE THREAD

**Issue by MassiveLag** [Crazy-Crew/BlockParticles#22]
I think this is because the fountain is too fast. Can you make blockparticle more optional? like adding interval for fountains & particles? I would like to put my fountain interval lower.

https://i.insanemc.net/brave_0TTRiaXSLC.png

**Comment by badbones69** [comment]
Yes, this is an issue that I have been looking into. I'll see what I can do with this info thanks.

**Comment by RyderBelserion** [comment]
Added to project board.
